### DISCUSSION TITLE
Allow for Powershell Scripts to be synced from Flow Designer Actions.

### DISCUSSION THREAD

**Discussion by Deathsnite** [arnoudkooi/SN-Utils#492] [Ideas]
Would be really nice to include the powershell scripts option from flow designer and have them sync as well.

**Comment by arnoudkooi** [comment]
DO you have a screenshot / details what you mean?

**Comment by Deathsnite** [comment]
<img width="1093" alt="image" src="https://github.com/arnoudkooi/ServiceNow-Utils/assets/50645484/80ba8f4a-36d8-4703-b05a-6f974f74a91f">

when i try to script sync a powershell script action instead of just a script action it tells me "No Script Steps Found". I would like it to take the command section and sync it into my VS Code like the other "scripts" actions do.

**Comment by arnoudkooi** [comment]
Why not :)

The updated versions are:
- SN Utils: 7.4.4.0 
- sn-scriptsync: 3.3.2 

Please upgrade both, and test to see if it works!

**Comment by Deathsnite** [comment]
Works great! Had to change the language mode on the VS Code side. But an easy enough fix. Thanks!
Just waiting on it to get to the Edge store as well.

**Comment by arnoudkooi** [comment]
You can install SN Utils from the Chrome Store in Edge, Edge usually takes a week.
It should save as a .ps1 file in VS code. Are you sure you have upgraded both to the latest version

**Comment by Deathsnite** [comment]
would help to update the VS Code side too... everything looks good. Thanks a ton!

**Comment by arnoudkooi** [comment]
Great, a review would be appreciated :)
